CHANGELOG_PENDING.md

```diff
@@ -6,19 +6,12 @@
 
 ### Features ⚒
 
-- \#2157 Add support for EIP-712 typed data signing in `livepeer_cli` (@yondonfu)
-
 #### General
 
-- \#2163 Fix Session crashes right after the O sends a tx, revert #2111 Ensure `maxFeePerGas` in all transactions never exceed `-masGasPrice` (@leszko)
```

Comment:
Some of the pending changes here are not being included on the official changelog. Is that by design?

Reply:
These entries were actually already copied into the [CHANGELOG](https://github.com/livepeer/go-livepeer/blob/master/CHANGELOG.md#bug-fixes-) for the v0.5.24 release, but I just forgot to clear CHANGELOG_PENDING so those entries stuck around and now are being removed.